tanner/config.py

```diff
@@ -11,7 +11,7 @@
                    'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
                    'EMULATORS': {'root_dir': '/opt/tanner'},
                    'SQLI': {'db_name': 'tanner.db'},
-                   'LOGGER': {'log_file': '/opt/tanner/tanner.log'},
+                   'LOGGER': {'log_debug': '/opt/tanner/tanner.log', 'log_err': '/opt/tanner/tanner.err'},
```

Comment:
I would leave debug logs as they are and provide it by default since most of the logs go to the debug logfile anyway.